docs/docs/user/grpc.md

```diff
@@ -12,15 +12,13 @@ This guide explains the functionalities of the `jelly-grpc` module, which implem
 
 As with the `jelly-stream` module, you can use `jelly-grpc` with any RDF library that has a Jelly integration, such as [Apache Jena](jena.md) (using `jelly-jena`) or [RDF4J](rdf4j.md) (using `jelly-rdf4j`). The gRPC API is generic and identical across all libraries.
 
-## Making a gRPC server and client
+[//]: # (## Making a gRPC server and client)
```

Comment:
Please note in #381 to uncomment this part.

Reply:
noted